src/utils/schema.js

```diff
@@ -54,45 +51,52 @@ module.exports.mergeMultiObjectFields = function (schema, objs) {
 	return model;
 };
 
-module.exports.slugifyFormID = function (schema, prefix = "") {
+const slugifyFormID = (schema, prefix = "") => {
 	// Try to get a reasonable default id from the schema,
 	// then slugify it.
 	if (typeof schema.id !== "undefined") {
 		// If an ID's been explicitly set, use it unchanged
 		return prefix + schema.id;
 	} else {
 		// Return the slugified version of either:
-		return prefix + (schema.inputName || schema.label || schema.model || "")
+		return (
+			prefix +
```

Comment:
Are you duplicating the `slugify` function in here?

Reply:
Again, it come from 17aba74, I have no idea why there is two slightly different slugify...